providers/github/github.go

```diff
@@ -51,26 +51,44 @@ func (p *Provider) GetTargetURL() (string, error) {
 func (p *Provider) addProductPath(product string) {
 	switch product {
 	case "dashboard":
-		var org string
-		if org = p.Ctx.String("org"); org != "" {
+		if org := p.Ctx.String("org"); org != "" {
 			p.join(fmt.Sprintf("orgs/%s/dashboard", org))
 			return
 		}
 		p.URL = p.baseURL
 		return
 	case "trending":
 		p.join("trending")
-		var since string
-		if since = p.Ctx.String("since"); since != "" {
+		if since := p.Ctx.String("since"); since != "" {
 			param := url.Values{}
 			param.Add("since", since)
 			p.URL.RawQuery = param.Encode()
 		}
-		var language string
-		if language = p.Ctx.String("language"); language != "" {
+		if language := p.Ctx.String("language"); language != "" {
 			p.join(language)
 			return
 		}
+	case "repository":
+
+		users := p.Ctx.String("users")
+		org := p.Ctx.String("org")
+		if users != "" && org != "" {
+			p.URL = p.baseURL
+			return
```

Comment:
You told me that we can return an `error` but `addProductPath` has not any type of variable.
So I changed that if both `users` and `org` was filled in, return base url.
I think we need discuss it if change interface: +1:

Reply:
> I think we need discuss it if change interface: +1:

Ahhhh, I see. Thank you for catching this. I agree with you.